web/src/js/components/header.js

```diff
@@ -227,32 +227,74 @@ var ViewMenu = React.createClass({
     mixins: [Router],
     toggleEventLog: function () {
         var d = {};
-
         if (this.getQuery()[Query.SHOW_EVENTLOG]) {
             d[Query.SHOW_EVENTLOG] = undefined;
         } else {
             d[Query.SHOW_EVENTLOG] = "t"; // any non-false value will do it, keep it short
         }
 
         this.updateLocation(undefined, d);
+        console.log('toggleevent');
     },
     render: function () {
-        var showEventLog = this.getQuery()[Query.SHOW_EVENTLOG];
-        return (
-            <div>
-                <button
-                    className={"btn " + (showEventLog ? "btn-primary" : "btn-default")}
-                    onClick={this.toggleEventLog}>
-                    <i className="fa fa-database"></i>
-                &nbsp;Show Eventlog
-                </button>
-                <span> </span>
-            </div>
-        );
+      var showEventLog = this.getQuery()[Query.SHOW_EVENTLOG];
+      return (
+        <div>
+          <ToggleComponent
+            checked={showEventLog}
+            name = "Show Eventlog"
+            onToggleChanged={this.toggleEventLog}
+            icon = "fa fa-database"/>
+        </div>
+      );
     }
 });
 
 
+class OptionMenu extends React.Component{
+    static title = "Options";
+
+    constructor(props){
+      super(props);
+      this.state = {
+        options :
+        [
+          {name: "--host", checked: true},
+          {name: "--no-upstream-cert", checked: false},
+          {name: "--http2", checked: false},
+          {name: "--anticache", checked: false},
+          {name: "--anticomp", checked: false},
+          {name: "--stickycookie", checked: true},
+          {name: "--stickyauth", checked: false},
+          {name: "--stream", checked: false}
+        ]
+      }
+    }
```

Comment:
I think the set of options we have is quite fixed. No need to put that in the state. :-)